objectscale-manager/templates/objectscale-manager-app-configmap.yaml

```
@@ -27,6 +27,16 @@ data:
           args:
             - -target-version
             - {{ default .Values.tag .Values.healthChecks.preUpdate.image.tag}}
+        - name: cloudiq-inventory-report
```

keep it as inventory in case we expand it to something else

```suggestion
        - name: cloudiq-inventory
```

---

Fixed